hack/jenkins/osx_integration_tests_virtualbox_systemd.sh

```diff
@@ -1,56 +0,0 @@
-#!/bin/bash
```

Comment:
Could you also drop the pending job here: https://github.com/kubernetes/minikube/blob/master/hack/jenkins/minikube_cross_build_and_upload.sh